### ISSUE TITLE
[Bug] Chat messages caught by AutoMod still trigger a normal chat event

### ISSUE THREAD

**Issue by TheBlackParrot** [crowbartools/Firebot#2425]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

https://www.twitch.tv/videos/2068356920?t=06h07m04s
"zeldasnowleopard" sends a message that gets caught by AutoMod, all mods see it, normal users (and my overlays) do not, but my TTS system (that runs through Firebot and its chat message events) reads it out as if AutoMod didn't catch it.

### Expected Behavior

The message in question does not trigger a normal chat message event

### Steps To Reproduce

1. Have someone not a mod send a message that will definitely trigger AutoMod detection (swear words with the highest AutoMod levels should trigger it)
2. See the chat message as if AutoMod didn't catch it

### Version

5.61.2

### What operating system are you using Firebot on?

Windows 10

### Contact Details (Optional)

theblackparrot

**Comment by CKY-** [comment]
this is a duplicate of #1640

**Comment by zunderscore** [comment]
This isn't exactly a duplicate. This is less about TTS and more about the Chat Message event triggering in an AutoMod scenario.

I was unable to reproduce this in our current dev branch. I setup a Chat Message event and turned on AutoMod settings, then sent a message from an alt account that AutoMod would catch. AutoMod did, in fact, catch the message and the Chat Message event did NOT trigger in Firebot until after I approved the message. I tested with another message, denied the AutoMod request, and the Chat Message event never triggered.

I'll leave this open until the next release just for validation.
